src/upload.test.ts

```diff
@@ -133,12 +134,30 @@ describe("uploadFile", () => {
     );
   });
 
+  it("should throw if 'file' input is not a file", async () => {
+    mock.resetHandlers();
+
+    // @ts-expect-error we only check this use case in case someone ignores typescript typing
+    await expect(client.uploadFile("some/path/file.json")).rejects.toThrowError(
+      "Expected parameter 'file' to be type 'File', was 'some/path/file.json'"
```

Comment:
nit: This error message will be better if it says "was type 'string', content 'some/path/file.json'".